pkg/apis/forklift/v1beta1/plan/vm.go

```diff
@@ -30,6 +30,8 @@ type VM struct {
 	ref.Ref `json:",inline"`
 	// Enable hooks.
 	Hooks []HookRef `json:"hooks,omitempty"`
+	// Choose the bootable disk number for praimery
+	MultiBootDiskNumber int `json:"multiBootDiskNumber,omitempty"`
```

Comment:
Should be just name I think, the number is an implementation detail for us. I think `RootDisk` or something would better represent this